### PR TITLE
docs/admin: document OpenID Connect plugin "extra-scopes" field

### PR DESCRIPTION
**Pull request by ericchiang**
Updates https://github.com/kubernetes/kubernetes/issues/37875

cc @kubernetes/sig-auth-pr-reviews

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3008)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -299,13 +299,14 @@ The first option is to use the `oidc` authenticator.  This authenticator takes y
...
-   --auth-provider=oidc
+   --auth-provider=oidc \
    --auth-provider-arg=idp-issuer-url=( issuer url ) \
    --auth-provider-arg=client-id=( your client id ) \
    --auth-provider-arg=client-secret=( your client secret ) \
    --auth-provider-arg=refresh-token=( your refresh token ) \
    --auth-provider-arg=idp-certificate-authority=( path to your ca certificate ) \
-   --auth-provider-arg=id-token=( your id_token )
+   --auth-provider-arg=id-token=( your id_token ) \
+   --auth-provider-arg=extra-scopes=( comma seperated list of scopes to add to "openid email profile", optional )
```
Comment (liggitt) [inline docs/admin/authentication.md]: separated

File: docs/admin/authentication.md [hunk 2]
```diff
@@ -299,13 +299,14 @@ The first option is to use the `oidc` authenticator.  This authenticator takes y
 
 ```bash
 kubectl config set-credentials USER_NAME \
-   --auth-provider=oidc
+   --auth-provider=oidc \
```
Comment (liggitt) [inline docs/admin/authentication.md]: nice

**Comment by liggitt** [comment]
nit on typo, LGTM otherwise

**Comment by chenopis** [comment]
@liggitt Is this good to go now?

**Comment by liggitt** [comment]
yes

**Pull request merged**